src/hooks/useBalanceChartData/useBalanceChartData.ts

```diff
@@ -235,10 +222,9 @@ export const calculateBucketPrices: CalculateBucketPrices = args => {
     // if we have txs in this bucket, adjust the crypto balance in each bucket
     txs.forEach(tx => {
       if (tx.fee && assetIds.includes(tx.fee.assetId)) {
-        // balance history being built in descending order, so fee means we had more before
-        // TODO(0xdef1cafe): this is awful but gets us out of trouble
-        // NOTE: related to utxo balance tracking, just ignoring bitcoin for now as our only utxo chain support
-        if (tx.chain !== KnownChainIds.BitcoinMainnet) {
+        // don't count fees for UTXO chains
+        if (fromChainId(tx.chain).chainNamespace !== CHAIN_NAMESPACE.Bitcoin) {
```

Comment:
charts would have been wrong for DOGE/LTC/BCH but we didn't notice

Reply:
You bet we did https://github.com/shapeshift/web/pull/2309#pullrequestreview-1060584975